public/printit.js

```diff
@@ -0,0 +1,52 @@
+window.PrintIt = (function(){
```

Comment:
For this whole thing, I think it would be more elegant to use FormData (if multipart/data is truly needed) and fetch. Looking at the stats of "can-i-use" it seems it's fairly safe to assume that most our apps/clients will use compatible browsers: https://caniuse.com/fetch

Reply:
I'll keep it for backward compatibility. That javascript is deprecated anyway but would allow a dropin replacement of printit 1.x by 2.x in various customer projects.